Document how to get a CLIENT_ID and CLIENT_SECRET from the local keyc… (#75)
project-kessel/docs: src/content/docs/building-with-kessel/how-to/migrate-from-rbac-v1-to-v2.mdx

	echo "Issuer URL: ${ISSUER_URL}"
	```

1. Get the subject (`sub` claim) for the service account:

	```shell
	SUB=$(curl -s ${ISSUER_URL}/protocol/openid-connect/token -d grant_type=client_credentials -d client_id=${CLIENT_ID} -d client_secret=${CLIENT_SECRET} | jq -r '.access_token' | awk -F. '{print $2}' | base64 --decode 2>/dev/null | jq -r '.sub')
	```

1. Configure RBAC to grant unlimited access to the service account:

	```shell
	oc create secret generic system-users --dry-run=client -o yaml --from-literal="system-users.json={
	  \"$SUB\": {
	    \"admin\": true,
	    \"is_service_account\": true,
	    \"allow_any_org\": true
	  }
	}" | oc apply -f -
	```

1. Enable token validation in RBAC:

	```shell
	oc get clowdapp rbac -o json | \
		jq '(.spec.deployments[] | select(.name=="service") | .podSpec.env[] | select(.name=="IT_BYPASS_TOKEN_VALIDATION") | .value) = "False"' | \
		oc apply -f -
	```

1. Update the Kessel Inventory service configuration to enable authentication.
   Run `oc edit cm inventory-api-config` to edit the kessel-inventory configuration ConfigMap and in the editor update the authn section like this:

	```yaml
	    authn:
	      oidc:

> 	        authn-server-url: https://${ISSUER_URL}/auth/realms/redhat-external

Author: ```suggestion
	        authn-server-url: ${ISSUER_URL}
```

> 	        authn-server-url: https://${KEYCLOAK_URL}/auth/realms/redhat-external

Author: ```suggestion
	        authn-server-url: https://${ISSUER_URL}/auth/realms/redhat-external
```

> 	        client-id: foo

Author: ```suggestion
	        client-id: ${CLIENT_ID}
```

	        skip-client-id-check: true
	        insecure-client: false
	```

> 	Note: Replace `${ISSUER_URL}` and `${CLIENT_ID} with the actual values.

Author: ```suggestion
	Note: Replace `${ISSUER_URL}` and `${CLIENT_ID}` with the actual values.
```

> 	Note: Replace `${KEYCLOAK_URL}` with the actual Keycloak URL from step 1.

Author: ```suggestion
	Note: Replace `${ISSUER_URL}` and `${CLIENT_ID} with the actual values.
```
